src/Test.UnitTests.Sarif/Baseline2/V2ResultMatcherTests.cs

```diff
@@ -199,5 +200,151 @@ public void V2ResultMatcher_WhenExistingSuppressedIssueIsNewlyUnsuppressed_DoesN
+                            new ReportingDescriptor { Id = TestConstants.RuleIds.Rule1 },
+                            new ReportingDescriptor { Id = TestConstants.RuleIds.Rule2 },
+                        }
+                    }
+                },
+                AutomationDetails = new RunAutomationDetails
+                {
+                    Guid = TestConstants.AutomationDetailsGuid
+                },
+                Results = new Result[]
+                {
+                    new Result
+                    {
+                        RuleId = TestConstants.RuleIds.Rule1,
+                        RuleIndex = 0
+                    },
+                    new Result
+                    {
+                        RuleId = TestConstants.RuleIds.Rule2,
+                        RuleIndex = 1
+                    }
+                },
+                Conversion = new Conversion
+                {
+                    Tool = new Tool
+                    {
+                        Driver = new ToolComponent
+                        {
+                            Name = TestConstants.ConverterName
+                        }
+                    }
+                },
+                Taxonomies = new ToolComponent[]
+                {
+                    new ToolComponent
+                    {
+                        Name = TestConstants.TaxonomyName,
+                        Taxa = new ReportingDescriptor[]
+                        {
+                            new ReportingDescriptor { Id = TestConstants.TaxonIds.Taxon1 },
+                            new ReportingDescriptor { Id = TestConstants.TaxonIds.Taxon2 },
+                        }
+                    }
+                },
+                Policies = new ToolComponent[]
+                {
+                    new ToolComponent
+                    {
+                        Name = TestConstants.PolicyName,
+                        Rules = new ReportingDescriptor[]
+                        {
+                            new ReportingDescriptor
+                            {
+                                Id = TestConstants.RuleIds.Rule2,
+                                DefaultConfiguration = new ReportingConfiguration { Level = FailureLevel.Error }
+                            }
+                        }
+                    }
+                },
+                Translations = new ToolComponent[]
+                {
+                    new ToolComponent
+                    {
+                        Name = TestConstants.TranslationName,
+                        TranslationMetadata = new TranslationMetadata
+                        {
+                            Name = TestConstants.TranslationMetadataName
+                        }
+                    }
+                },
+                Language = TestConstants.LanguageIdentifier,
+                RedactionTokens = new string[]
+                {
+                    TestConstants.RedactionToken
```

Comment:
>TestConstants.RedactionToken [](start = 20, length = 28)

If you have time to search the source for 'REDACTED' and ensure you are happy with how DRY our world is around this constant, would be a nice to have.  I see 6 instance to review.

Reply:
I only see three occurrences:

- The existing `SarifConstants.RedactedMarker`, which I forgot we had,
- The new `TestConstants.RedcationToken`, which I now see we don't need and have removed.
- A hard-coded occurrence in a single test in `SarifValidatorTests`, which I've fixed.

So now it's all cleaned up unless I missed something.

---
In reply to: [328782431](https://github.com/microsoft/sarif-sdk/pull/1686#discussion_r328782431) [](ancestors = 328782431)